helm/cluster-gcp/templates/_bastion.tpl

```diff
@@ -37,13 +25,17 @@ spec:
         {{- include "labels.common" $ | nindent 8 }}
     spec:
       bootstrap:
-        dataSecretName: {{ include "resource.default.name" $ }}-bastion-ignition
+        configRef:
+          apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+          kind: KubeadmConfigTemplate
+          name: {{ include "resource.default.name" $ }}-bastion
       clusterName: {{ include "resource.default.name" $ }}
+      failureDomain: {{ index .Values.gcp.failureDomains 0 }}
       infrastructureRef:
         apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
         kind: GCPMachineTemplate
         name: {{ include "resource.default.name" $ }}-bastion
-      version: v0.0.0
+      version: {{ .Values.kubernetesVersion }}
```

Comment:
If I understand correctly, the k8s version here doesn't really matter right?

Reply:
Yeah, previously we used `0.0.0` but that is not in this case allowed, the k8s will never launch on bastion